python/ray/serve/backend_worker.py

```diff
@@ -82,24 +79,11 @@ def __init__(self, name, _callable, router_handle, is_function):
         self.router_handle = router_handle
         self.is_function = is_function
 
-        self.error_counter = 0
-        self.latency_list = []
-
-    def get_metrics(self):
-        # Make a copy of the latency list and clear current list
-        latency_list = self.latency_list[:]
-        self.latency_list = []
-
-        return {
-            "{}_error_counter".format(self.name): {
-                "value": self.error_counter,
-                "type": "counter",
-            },
-            "{}_latency_s".format(self.name): {
-                "value": latency_list,
-                "type": "list",
-            },
-        }
+        self.metric_collector = PushCollector.connect_from_serve()
+        self.error_counter = self.metric_collector.new_counter(
+            "backend_error_counter",
+            description="Number of exceptions occurred in backend",
+            labels={"backend": self.name})
```

Comment:
I think self.name is a replica tag, right? We should probably change the key here to `replica` or `replica_tag`